readme.md

```diff
@@ -102,6 +102,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [View linked gists inline in comments.](https://user-images.githubusercontent.com/6978877/33911900-c62ee968-df8b-11e7-8685-506ffafc60b4.PNG)
 - [Avoid opening duplicate issues thanks to the list of possibly-related issues.](https://user-images.githubusercontent.com/29176678/37566899-85953e6e-2abf-11e8-9f0e-52d18c87bbe3.gif)
 - [Use the pull request description as the commit message when merging with `Squash and merge`](https://github.com/sindresorhus/refined-github/issues/1322).
+- [View follower status on user profiles.](https://user-images.githubusercontent.com/3723666/44477924-1a8afc00-a60a-11e8-9528-2ad75958b3a9.gif)
```

Comment:
This could be written more clearly:

> Badge on a user's profile if they follow you.

Reply:
This should be in the “More info at a glance” section.